code/modules/research/designs/wiremod_designs.dm

```diff
@@ -32,7 +32,7 @@
 /datum/design/component
 	name = "Component ( NULL ENTRY )"
 	desc = "A component that goes into an integrated circuit."
-	build_type = IMPRINTER | COMPONENT_PRINTER
+	build_type = IMPRINTER | COMPONENT_PRINTER | COMPONENT_INTEGRATED
```

Comment:
do you want these to be printable normally? seems like a waste of materials.

Reply:
I have considered removing them from the lathe, I just think it might be worth leaving them there for familiarity; but yeah, those would be long-term deprecated.